README.md

```diff
@@ -1,16 +1,27 @@
-# cup-of-Ticket
+# Cup of Tickets
 
-Frontend URL: 
-https://orange-pond-040622003.2.azurestaticapps.net
+Project made by [Lidia Orfanu](https://github.com/LidiaOrfanu), [Luc Siat](https://github.com/Luc-Siat), [Donna Soujeh](https://github.com/proksimacentauri), [Caspar Rödöö](https://github.com/cparn) in a week and a half.
 
-Backend URL:
-https://cup-of-tickets-backend.azurewebsites.net
+[Frontend URL](https://orange-pond-040622003.2.azurestaticapps.net)
+[Backend URL](https://cup-of-tickets-backend.azurewebsites.net)
 
```

Comment:
Change frontend to deployed app.
Either remove backend or change to swagger, just having the api url make no sense